code/modules/mob/living/carbon/xenomorph/Castes/Necromorph.dm

```diff
@@ -0,0 +1,47 @@
+/datum/xeno_caste/necromorph
+	caste_name = "Necromorph"
+	display_name = "Necromorph"
+	upgrade_name = ""
+	caste_desc = "A reinimated iteration of a human, looks like a hackjob."
+	caste_type_path = /mob/living/carbon/Xenomorph/Necromorph
+	tier = 0
+	upgrade = 0
+
+	// *** Melee Attacks *** //
+	melee_damage_lower = 20
+	melee_damage_upper = 30
+	attack_delay = -2
+
+	// *** Tackle *** //
+	tackle_damage = 30
+
+	// *** Speed *** //
+	speed = -1
+
+	// *** Plasma *** //
+	plasma_max = 50 // can't be zero, causes runtimes
```

Comment:
In the long term this would likely not be a xeno caste but a carbon subtype. But for now this will have to do with all the hacky solutions that go along with it.

Reply:
Yup, discussed it with rohesie for such.